analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.Mutex.cs

```diff
@@ -23,8 +23,6 @@ public void Noncompliant(Foo foo)
             var m3 = Mutex.OpenExisting("x");
             m3.WaitOne(); // Noncompliant
 
-            var m4 = new Mutex();
```

Comment:
Without this removal, the test fails. This is likely caused by the new branching and the explosion of states.